spec/jobs/education_form/create_daily_fiscal_year_to_date_report_spec.rb

```diff
@@ -78,6 +79,8 @@ def get_education_form_fixture(filename)
       create(:education_benefits_submission, form_type: '0993', created_at: date, region: :western)
       create(:education_benefits_submission, form_type: '0994',
                                              created_at: date, region: :eastern, vettec: true, chapter33: false)
+
+      # create(:education_benefits_submission, form_type: 'FAKE', created_at: date)
```

Comment:
Just double checking to see if you meant to include this commented out line in there.

Reply:
Thanks, I did not!